### PR TITLE
DOC-4226 Add a step about restarting server

### PR DESCRIPTION
**Pull request by clairecadman**
https://tickets.puppetlabs.com/browse/DOC-4226

### CODE REVIEW

**Comment by mwaggett** [comment]
Is this the right target branch? I didn't even know `6.6.0` was a thing..?

**Comment by Magisus** [comment]
It shouldn't be... this probably should be targeted at 5.3.x.

File: documentation/gems.markdown [hunk 1]
```diff
@@ -88,6 +88,8 @@ settings with Puppet. You can manage Puppet Server's gem dependencies with the
 package provider shipped in [`puppetlabs-puppetserver_gem`](https://forge.puppet.com/puppetlabs/puppetserver_gem)
 module.
 
+>Note: If you try to load a gem before it's been installed, the agent run will fail with a `LoadError`.
```
Comment (Magisus) [inline documentation/gems.markdown]: It seems like the important part of that ticket was the note that the server needs to be restarted after installing the gem. This note might also be useful, but I notice we also don't mention the actual solution to his problem anywhere in this section.

**Comment by Magisus** [comment]
It might be easier to make a new PR based against that branch. The git UI doesn't handle retargeting from newer branches to older ones well...

**Pull request closed without merging**